src/gui/mzroll/videoplayer.cpp

```diff
@@ -0,0 +1,162 @@
+#include "videoplayer.h"
+
+#include <QtWidgets>
+#include <QVideoWidget>
+#include <QSizePolicy>
+#include <QSettings>
+#include <QPushButton>
+
+VideoPlayer::VideoPlayer(QSettings *settings,QWidget* parent)
+    : QWidget(parent)
+    , m_settings(settings)
+{
+    resize(750,700);
+    QBoxLayout *layout = new QVBoxLayout(this);
+    layout->setContentsMargins(20,20,20,20);
+    m_title = new QLabel(this);
+    m_title->setText("<b>Make your analyses more insightful with ML. View your fluxomics workflow in PollyPhi</b>");
+    m_title->setContentsMargins(0,0,0,10);
+    m_title->setAlignment(Qt::AlignCenter);
+    layout->addWidget(m_title);
+
+    m_message = new QLabel(this);
+    m_message->setWordWrap(true);
+    m_message->setAlignment(Qt::AlignCenter);
+    m_message->setContentsMargins(0,0,0,20);
+    m_message->setText("Classify groups as Good and Bad quickly and with accuracy with the new Machine Learning \
+algorithm. Save time classifying data and spend more time analyzing it. ");

```

Comment:
Maybe we should not be setting such message strings in the constructor itself. We might need this player later for other demonstrations as well. But it need not be changed right now.

Reply:
Ya, this should be used as a generic video player. Will change this if we add another video